add-on/panels/toc.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/.  -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <base href="chrome://loop/content">
+    <link rel="stylesheet" type="text/css" href="shared/css/reset.css">
```

Comment:
Please add an XXX to review all these inclusions once the dust settles a bit. I don't want us to include unnecessary css/js files.